contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
```

Comment:
Well Git's man pages are pretty dense, but the one you mentioned doesn't seem that bad.

On March 29, 2015 5:35:04 PM EDT, "Eliseo Martínez" <notifications@github.com> wrote:
>> @@ -0,0 +1,41 @@
>> +## What's this?
>> +
>> +A basic set of tools to ease adding review trailers, in order to
>acknowledge reviewers' work.
>
>Ok. Maybe http://git-scm.com/docs/git-interpret-trailers better?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/2291/files#r27359403